### DISCUSSION TITLE
How to make original image uploaded a certain size

### DISCUSSION THREAD

**Discussion by jimoheokoh** [spatie/laravel-medialibrary#2581] [Q&A] [answered]
I notice currently if you want to reduce the size of the uploaded images, you use conversions.
What if I don't want to use conversions but keep original image to a certain size and transform png to jpg all without conversions

Is this possible?

**Comment by freekmurze** [comment, marked as answer]
In that case you'll need to transform the image yourself before adding it to the media library.
